senior-knights/course-schedulizer
# Edit multiple meetings support

To Test:
- Notice that there is only one export button now (for the full/draft version of the csv)
- Create a section
- Click that section, and change some subset of the start time, duration, days, and/or room, then click `Add New Section`
- Now export and notice that there is only one entry in the CSV (which contains both meetings (or meeting sets might be a more accurate term))
- Try editing one of the meetings, export again, and ensure that your changes only applied to the meeting that you clicked on
- Click `Delete Section` for one of the meetings, and ensure that it only deleted that meeting (then you can try deleting the other, and that should delete the course and section itself)
- Try out other edge cases, like adding a meeting for an existing section in the math schedule, updating a course/section like normal, non-teaching loads still work, etc.

Notes:
- This is a pretty major change to how our app works, so it should be treated with some caution in testing
- Is it confusing that pressing `Delete Section` won't delete the whole section if there are multiple meetings in that section? Should we call this button, `Delete Meeting` or something else?
- This will unfortunately make it so that changes which apply on the section level must be made for each meeting in the section individually (so if you had 2 meetings for a section taught by Bob and wanted to change the section to be taught by Sarah, then you would need to do it for both meetings). We may want to brainstorm ways to make section/course-wide edits in the next meeting.

Closes #139
Also Closes #157
